src/commands/Fun/markov.ts

```diff
@@ -35,14 +33,17 @@ export default class extends SkyraCommand {
 	private kBoundUseUpperCase!: (wordBank: WordBank) => string;
 	private kProcess!: (message: GuildMessage, language: TFunction, markov: Markov) => Promise<MessageEmbed>;
 
-	public async run(message: GuildMessage, [channnel, username]: [TextChannel?, User?]) {
-		const t = await message.fetchT();
+	public async run(message: GuildMessage, args: SkyraCommand.Args) {
+		const channel = args.finished
+			? (message.channel as TextChannel)
+			: await args.pick('textChannelName').catch(() => message.channel as TextChannel);
+		const username = args.finished ? message.author : await args.pick('userName');
```

Comment:
I'd call consistency with the one on top, but this IS cleaner... we really need rust-like Results 🤣